pkg/proto/utils/tagger.go

```diff
@@ -0,0 +1,87 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package utils
```

Comment:
nit: Mixing a util into a `pkg/proto` package seems unnecessary and potentially bad for maintenance - why not move this to `pkg/tagger` (or `pkg/tagger/util/`)?

Reply:
We could try that, but I think having these here avoids the likelihood of dependency cycles; something which also motivated this PR.